toucan_connectors/toucan_connector.py

```diff
@@ -32,8 +33,6 @@
 
 
 class DataStats(BaseModel):
-    total_rows: Optional[int] = None
-    total_returned_rows: Optional[int] = None
```

Comment:
Sooo coul, we should probably get rid of those values also here : https://github.com/ToucanToco/http_data_service/blob/ee3a8d058f9427e7a954d6e8e5c57ea952a6c22a/src/data_services_common/models.py#L31 

also... thinking... it could have been great to have our models shared amount repos... so that this kind of change can be done only once somewhere... and just bumped everywhere,,, ? WDYT @lukapeschke ?